README.md

```diff
@@ -96,18 +96,44 @@ These values can be found in the “Settings” tab within the Auth0 application
 
 ### Calling the Deploy CLI
 
-Finally, with above complete, the Deploy CLI export command can be run:
+Finally, with above complete, the Deploy CLI `export` command can be run, here using environemt variables:
 
 ```shell
 a0deploy export --format=yaml --output_folder=local
 ```
 
-Once the process completes, observe the resource configuration files generated in the `local` directory. Then, run the import command, which pushes configuration from the local machine to your Auth0 tenant:
+Once the process completes, observe the resource configuration files generated in the `local` directory. Then, run the `import` command, which pushes configuration from the local machine to your Auth0 tenant, here using a `config_file` instead of environment variables:
 
 ```shell
-a0deploy import -c=config.json --input_file local/tenant.yaml
+a0deploy import --config_file=config.json --input_file local/tenant.yaml --env=false
```

Comment:
The behavior of environment variables is made clear in this section: https://github.com/auth0/auth0-deploy-cli/blob/master/docs/configuring-the-deploy-cli.md#environment-variables. 

Because the README is meant to be more of a starting point, I'd rather not introduce concepts outside whats necessary to getting started.

Reply:
This could be put back - I was under the impression that if you used the `config_file` then you would also want to set `--env=false`.